src/app/campaign/store/models/campaign-state.factory.ts

```diff
@@ -80,7 +128,7 @@ export const allocationPreviewParamsFixture = {
   endAt: flightFixture.endAt,
   name: flightFixture.name,
   set_inventory_uri: flightFixture.set_inventory_uri,
-  zones: flightFixture.zones,
+  zones: flightFixture.zones.map(zone => ({ id: zone.id })),
```

Comment:
Removing the labels from these?  Or are there other properties messing things up?  (Does seem like per PRX/augury.prx.org#206 this can someday just be the Flight model).

Reply:
Yeah... tbh I do not love what happened to the allocation preview params zone with that recent zone mp3 url stuff. You changed that zone param to an object with just the id, so I had to make some adjustments in a few places, was thinking about changing it back to just the id string. https://github.com/PRX/grove.prx.org/blob/master/src/app/campaign/store/reducers/allocation-preview.reducer.ts#L15

Reply:
Yeah, I was wiring things through as if 206 was already done, until I realized that I couldn't just serialize the entire Flight model as-is.  (And never undid that change).  I'm fine with changing back to array-of-strings until 206 hits.

Reply:
ah ok if that preview endpoint is gonna take all those params then let's leave an an object and we'll sort it out later